lib/metadata/bucketclient/backend.js

```diff
@@ -27,75 +35,144 @@ class BucketClientInterface {
             });
             this.client = new bucketclient.RESTClient(bootstrap, log);
         }
+        this.init();
+    }
+
+    init() {
+        // get metadata information
+        const beginPath = '/default/metadataInformation/foo';
```

Comment:
will this path always be used?  It probably makes more sense to put the raw get request in bucketclient and have a wrapper function here.